Controllers/StockController.cs

```diff
@@ -0,0 +1,282 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Text.Json;
+using InvestingOak.Converters;
+using InvestingOak.Models.Finnhub;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InvestingOak.Controllers
+{
+    [Route("api/stocks")]
+    [ApiController]
+    [Produces("application/json")]
+    [ResponseCache(VaryByHeader = "User-Agent", Duration = 30)]
+    public class StockController : ControllerBase
+    {
+        private const string AlphaVantageKey = "HNAFS0NKXJHEG5YI";
+        private const string AlphaVantageBaseUrl = "https://www.alphavantage.co/query";
+        private const string FinnhubKey = "btt5puv48v6q0kg1m610";
+        private const string FinnhubBaseUrl = "https://finnhub.io/api/v1";
+        private readonly HttpClient alphaVantageClient;
```

Comment:
Codacy found an issue: [Remove this unread private field 'alphaVantageClient' or refactor the code to use its value.](https://app.codacy.com/gh/InvestingOak/InvestingOak/pullRequest?prid=6380270)